designer/test/helpers/sub-component-assertions.js

```diff
@@ -0,0 +1,17 @@
+import { Input } from "@govuk-jsx/input";
```

Comment:
Please push this file is Typescript? Types here can be more loose and it is fine to type cast.